### PR TITLE
support for cinnamon added

### PR DESCRIPTION
**Pull request by Rafi993**
support for cinnamon added

### CODE REVIEW

File: lib/linux.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 'use strict';
 const path = require('path');
+
 const childProcess = require('child_process');
+
 const pify = require('pify');
+
```
Comment (SamVerschueren) [inline lib/linux.js]: It should be like this

```js
const path = require('path');
const childProcess = require('child_process');
const pify = require('pify');

const cp = pify(childProcess);
```

Same goes for `win.js` and `macos.js`. A newline has to be added after the `require` statements at [win.js#L5](https://github.com/sindresorhus/wallpaper/blob/master/lib/win.js#L5) and [macos.js#L5](https://github.com/sindresorhus/wallpaper/blob/master/lib/macos.js#L5). That will fix this linting issues with travis.

**Comment by SamVerschueren** [comment]
Also change `.travis.yml` and `appveyor.yml` to run on `6` instead of `5`. So only `6` and `4`.

**Comment by sindresorhus** [comment]
Thank you :)

**Comment by kernelkiller** [comment]
I think this addition might have broken the wallpaper changes for the normal gnome desktop. Since both share "gsettings" as command, it replaces the old entry within the "availableAppsDict". In the end there are two identical entries within "availableApps" that reference the cinnamon command.

I figured this out while i investigated why your module doesn't work on my Ubuntu MATE image for the Raspberry Pi. In that case gsettings, gconftool-2, and dconf are available, but only dconf seems to be able to change the background. gsettings is the first found command with a setter within the "availableApps", but doesn't do anything - not even when it uses 'org.gnome.desktop.background' as key.

But i'm a little bit out of practice with Linux, so i'm not sure, if i did something wrong.

**Pull request closed without merging**